example_env_file

```diff
@@ -52,3 +52,5 @@ WEEK_ND=5
 
 HOST_UID=1000
 HOST_GID=1000
+
+ACTIVE_STATES = ['CE', 'MA', 'MG', 'RS', 'RJ', 'SC', 'SP', 'PR']
```

Comment:
it seems that it is missing an empty line at the end of the file.
you can configure your IDE to do it automatically for you.